tools/ci_build/build.py

```diff
@@ -1514,6 +1534,18 @@ def main():
     if args.skip_tests:
         args.test = False
 
+    if (args.include_ops_by_model and len(args.include_ops_by_model) > 0) or\
+       (args.include_ops_by_file and len(args.include_ops_by_file) > 0):
+
+        from provider_rewriter import rewrite_providers
+
+        include_ops_by_model = args.include_ops_by_model if args.include_ops_by_model else ''
+        include_ops_by_file = args.include_ops_by_file if args.include_ops_by_file else ''
+
+        rewrite_providers(include_ops_by_model, include_ops_by_file, paths_to_execution_provider(args))
```

Comment:
Is exclude_ops_in_providers a better name as I don't think there's any other reason to rewrite the provider files?

Reply:
Let's go with disable_ops_in_providers then.

Reply:
Done.